packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -518,6 +527,7 @@ export const BSC_TOKENS = {
     pBTC: '0xed28a457a5a76596ac48d87c0f577020f6ea1c4c',
     nUSD: '0x23b891e5c62e0955ae2bd185990103928ab817b3',
     BSW: '0x965F527D9159dCe6288a2219DB51fc6Eef120dD1',
+    WOO: '0x4691937a7508860f876c9c0a2a617e7d9e945d4b',
```

Comment:
Is `WOO` token address used?

Reply:
It is a tradeable token on woofi, so we should add it right?